nixos/modules/services/networking/tetrd.nix

```diff
@@ -1,7 +1,7 @@
 { config, lib, pkgs, ... }:
 
 {
-  options.services.tetrd.enable = lib.mkEnableOption pkgs.tetrd.meta.description;
+  options.services.tetrd.enable = lib.mkEnableOption "tetrd";
```

Comment:
If we wanted to keep that we'd need to set `meta.buildDocsInSandbox = false` fwiw.

Reply:
I would suggest to change to the service name here (or something else short). I think this is more readable.  In this particular case, it would also evaluate to: `Whether to enable the  Share your internet connection from your device to your PC and vice versa through a USB cable`, which seems a bit lengthy.